pkg/reconciler/taskrun/hostpool.go

```diff
@@ -42,9 +42,6 @@ func (hp HostPool) Allocate(r *ReconcileTaskRun, ctx context.Context, tr *v1.Tas
 			hostCount[host] = hostCount[host] + 1
 		}
 	}
-	for k, v := range hostCount {
-		log.Info("host count", "host", k, "count", v)
-	}
```

Comment:
This is duplicated below in line 67 with more useful information of free slots per host.